addon/components/file-dropzone/component.js

```diff
@@ -185,6 +187,9 @@ export default BaseComponent.extend({
         this[DATA_TRANSFER] = null;
       }
 
+      if (this.isDestroyed) {
+        return;
+      }
```

Comment:
Tests occasionally failed with `Cannot call set on destroyed object`.

Reply:
I think we need to use `this.get()` in order to support Ember 2.18, isn't it?

Reply:
You're right. I keep forgetting about `get`.

Fixed in d89dcb3 😅